datasetinsights/estimators/faster_rcnn.py

```diff
@@ -33,44 +36,102 @@ class FasterRCNN(Estimator):
     """
     Faster-RCNN train/evaluate implementation for object detection.
 
-    https://github.com/pytorch/vision/tree/master/references/detection
-    https://arxiv.org/abs/1506.01497
+
     Args:
         config (CfgNode): estimator config
-        writer: Tensorboard writer object
-        kfp_writer: KubeflowPipelineWriter object to write logs
-        checkpointer: Model checkpointer callback to save models
-        device: model training on device (cpu|cuda)
         box_score_thresh: (optional) default threshold is 0.05
-        gpu (int): (optional) gpu id on which code will execute
-        rank (int): (optional) rank of process executing code
+        distributed: whether or not the estimator is distributed
+        data_root: Root directory on localhost where datasets are located
+        kfp_metrics_filename: Kubeflow Metrics filename
+        kfp_metrics_dir: Path to the directory where Kubeflow
+         metrics files are stored
+
+
+    https://github.com/pytorch/vision/tree/master/references/detection
+    https://arxiv.org/abs/1506.01497
+
     Attributes:
         model: pytorch model
         writer: Tensorboard writer object
         kfp_writer: KubeflowPipelineWriter object
         checkpointer: Model checkpointer callback to save models
         device: model training on device (cpu|cuda)
+        gpu (int): (optional) gpu id on which code will execute
+        rank (int): (optional) rank of process executing code
     """
 
     def __init__(
         self,
         *,
         config,
-        writer,
-        kfp_writer,
-        checkpointer,
-        device,
+        logdir,
         box_score_thresh=0.05,
-        gpu=0,
-        rank=0,
+        data_root=None,
```

Comment:
Why logdir and data_root is not part of config?

Reply:
The config.system variables have been removed from the config. The model config does not include this data.